controller/search.js

```diff
@@ -21,6 +21,11 @@ function setup( backend, query ){
       cmd.type = req.clean.layers;
     }
 
+    // set type if input suggests targeting a layer(s)
```

Comment:
Can you think of any easy ways to make the logic for selecting target layers less brittle? Right now it relies on a flag that's set pretty far away, and then there are at least four different values here that determine the final value of cmd.type. lots of paths to think about when making future changes. Even if it's something we don't implement today, but a refactoring we work towards, i'd love to hear some thoughs.